crates/common/tedge_config/src/system_services/managers/config.rs

```diff
@@ -4,10 +4,12 @@ use std::fs;
 use std::path::PathBuf;
 
 pub const SERVICE_CONFIG_FILE: &str = "system.toml";
+const INIT_COMMAND: &[&str] = &["init", "6"];
 
 #[derive(Deserialize, Debug, Default, Eq, PartialEq)]
 pub struct SystemConfig {
-    pub(crate) init: InitConfig,
+    pub(crate) init: Option<InitConfig>,
```

Comment:
Why this change? Why not simply use `InitConfig::default()` when no config is provided?